c7n/resources/route53.py

```diff
@@ -481,7 +482,11 @@ def process(self, resources, event=None):
             logging = zid in enabled_zones
             if logging and state:
                 r['c7n:log-config'] = enabled_zones[zid]
+                log_group_name = r['c7n:log-config']['CloudWatchLogsLogGroupArn'].split(":")[-1]
+                response = cw_client.describe_subscription_filters(logGroupName=log_group_name)
```

Comment:
this new api call doesn't appear in the recorded data, is this functionalityy being tested?

Reply:
@kapilt i believe proper unit test has been added.